pkg/controllers/dynakube/processmoduleconfigsecret/conditions.go

```diff
@@ -1,5 +1,5 @@
 package processmoduleconfigsecret
 
 const (
-	pmcConditionType = "ProcessModuleConfig"
+	PmcConditionType = "ProcessModuleConfig"
```

Comment:
```suggestion
	PMCConditionType = "ProcessModuleConfig"
```
or `PMCconditionType`

Reply:
done https://github.com/Dynatrace/dynatrace-operator/pull/4515/commits/0b56b11c90782d9b28b1b631fe69a4e1d871a68e